types/topology.go

```diff
@@ -426,7 +426,7 @@ func (t *Topology) GetSANs(name string) []string {
 	return nil
 }
 
-// Returns the 'extras' section for the given node.
+// GetNodeExtras Returns the 'extras' section for the given node.
```

Comment:
thanks for fixing the comments patter
could you also decapitalise the next word after the method name?

Reply:
done!